homework/nwd-nnw/nwdNww.hpp

```diff
@@ -1,11 +1,22 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+   int temp;
+
+   while (rhs!=0) {
+       temp = rhs;
+       rhs = lhs%rhs;
+       lhs = temp;
+   }
+   if (lhs<0){
+       return lhs*(-1);
+   }
+   return lhs;
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if (lhs == 0 || rhs == 0) {
+        return 0;
+    }
+    return abs(lhs*rhs/NWD(lhs, rhs));
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..